music21/languageExcerpts/instrumentLookup.py

```diff
@@ -15,169 +16,141 @@
-    'acc.': 'accordion',
     'accdn': 'accordion',
-    'arp.': 'harp',
+    'arp': 'harp',
     'b dr': 'bass drum',
-    'b.': 'bass',
-    'b.cl.': 'bass clarinet',
-    'b.dr.': 'bass drum',
+    'bcl': 'bass clarinet',
+    'b cl': 'bass clarinet',
     'bag': 'bagpipes',
     'bar': 'baritone',
     'bar sax': 'baritone saxophone',
-    'bar.': 'baritone',
     'bgo dr': 'bongo drums',
-    'bj.': 'banjo',
+    'bj': 'banjo',
     'bjo': 'banjo',
-    'bkl.': 'bass clarinet',
-    'bn.': 'bassoon',
-    'br.': 'viola',
+    'bkl': 'bass clarinet',
+    'bn': 'bassoon',
+    'br': 'viola',
     'bs': 'bassoon',
     'bs cl': 'bass clarinet',
-    'bssn.': 'bassoon',
-    'c.c.': 'snare drum',
+    'bsn': 'bassoon',
+    'bssn': 'bassoon',
+    'cbsn': 'contrabassoon',
+    'c c': 'snare drum',
+    'c bsn': 'contrabassoon',
     'cas': 'castanets',
-    'cas.': 'castanets',
     'casts': 'castanets',
     'cb': 'contrabass',
-    'cb.': 'contrabass',
-    'cel.': 'celesta',
+    'cel': 'celesta',
     'cga dr': 'conga drum',
     'cl': 'clarinet',
-    'cl.': 'clarinet',
     'clst': 'celesta',
     'clv': 'clavichord',
-    'clvd.': 'clavichord',
-    'cor.': 'horn',
-    'cr.tr.': 'bass drum',
+    'clvd': 'clavichord',
+    'cor': 'horn',
+    'cor ang': 'english horn',
+    'cr tr': 'bass drum',
     'cwb': 'cowbells',
     'cym': 'crash cymbals',
-    'e. hn.': 'english horn',
-    'e.gtr.': 'electric guitar',
-    'e.h.': 'english horn',
+    'e hn': 'english horn',
+    'e gtr': 'electric guitar',
+    'e h': 'english horn',
     'elec b': 'electric bass',
     'elec gtr': 'electric guitar',
     'elec org': 'electric organ',
     'eng hn': 'english horn',
-    'eng.hn.': 'english horn',
-    'fg.': 'bassoon',
+    'fag': 'bassoon',
+    'fg': 'bassoon',
     'fing cym': 'finger cymbals',
-    'fing.cym.': 'finger cymbals',
     'fl': 'flute',
-    'fl.': 'flute',
-    'g.c.': 'bass drum',
-    'glck.': 'glockenspiel',
-    'glock.': 'glockenspiel',
-    'glsp.': 'glockenspiel',
+    'g c': 'bass drum',
+    'glck': 'glockenspiel',
+    'glock': 'glockenspiel',
+    'glsp': 'glockenspiel',
     'gng': 'gong',
-    'gr.cassa.': 'bass drum',
+    'gr cassa': 'bass drum',
     'gsp': 'glockenspiel',
-    'hb.': 'oboe',
+    'hb': 'oboe',
     'hmca': 'harmonica',
     'hn': 'horn',
-    'hn.': 'horn',
     'hp': 'harp',
-    'hp.': 'harp',
-    'hpd.': 'harpsichord',
-    'hpe.': 'harp',
+    'hpd': 'harpsichord',
+    'hpe': 'harp',
     'hpschd': 'harpsichord',
-    'hrp.': 'harp',
-    'k. dr.': 'timpani',
+    'hrp': 'harp',
+    'k dr': 'timpani',
     'kal': 'kalimba',
-    'kas.': 'castanets',
-    'kl.': 'clarinet',
-    'mand.': 'mandolin',
+    'kas': 'castanets',
+    'kl': 'clarinet',
+    'mand': 'mandolin',
     'mar': 'marimba',
-    'mar.': 'marimba',
     'mdln': 'mandolin',
     'mez': 'mezzo-soprano',
-    'mez.': 'mezzo-soprano',
-    'mezz.': 'mezzo-soprano',
+    'mezz': 'mezzo-soprano',
     'ob': 'oboe',
```

Comment:
one vote for `'mz'` while we're here :-)

Reply:
Fine by me!